src/scripts/cart.count.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const shoppingCartCounter = document.querySelector('.cart-counter');
+
+// Opcja z ingerencją w style CSS, określająca max. wskaźnik
+if (shoppingCartCounter.innerText > 99999) {
+  shoppingCartCounter.innerHTML = 99999;
+}
+
+/* Opcja w pełni JS, bez ingerencji w style CSS
+function adjustCartCounterDisplay () {
+  let shoppingCartItemsCount = shoppingCartCounter.innerText.length;
+
+  if (shoppingCartItemsCount < 4) {
+    shoppingCartCounter.style.fontSize = '13px';
+  } else if (shoppingCartItemsCount > 3 && shoppingCartItemsCount < 5) {
+    shoppingCartCounter.style.fontSize = '9px';
```

Comment:
unikaj manipulacji stylami bezpośrednio w JSie, zwłaszcza jeśli ingerujesz w style zdefiniowane również w CSS. lepszym pomysłem w tym wypadku byłoby ustawienie, w zależności od liczby znaków, odpowiedniej klasy dla elementu, którą następnie definiujesz w CSSach.

generalnie - im prościej, tym lepiej, dlatego opcja nr 1 wygrywa. to rozwiązanie w obecnej formie wygląda na przekombinowane.

Reply:
Ok, usunięty zakomentowany kod. Faktycznie, lepszą opcją byłoby manipulowanie dodawaniem klas.